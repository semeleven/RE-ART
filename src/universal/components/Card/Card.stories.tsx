import * as React from 'react';

import { storiesOf } from '@storybook/react';

import { Card } from '../../components';

storiesOf('Card', module).add('card', () => <Card>CARD</Card>);
