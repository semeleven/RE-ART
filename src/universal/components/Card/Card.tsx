// import styled from '../../lib/styled';
