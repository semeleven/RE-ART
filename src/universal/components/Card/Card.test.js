import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Card from './Card';

it('renders Card component', () => {
	const wrapper = shallow(<Card>Content</Card>);
	expect(wrapper).toMatchSnapshot();
});
