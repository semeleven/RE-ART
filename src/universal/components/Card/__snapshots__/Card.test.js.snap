// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Card component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: 'Lato',sans-serif;
  font-weight: 600;
  outline: none;
  font-size: 15px;
  line-height: 1;
  cursor: pointer;
  width: 100%;
  background-color: #ffffff;
  color: #16111C;
  padding: 40px;
  border: 1px solid #16111C;
  -webkit-transition: 0.2s ease-in-out;
  transition: 0.2s ease-in-out;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  margin-bottom: 20px;
}

.c0:hover {
  background-color: #16111C;
  color: #ffffff;
}

<div
  className="c0"
>
  Content
</div>
`;
